backend/db/migrate/20230203180846_fix_column_name_in_courses.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class FixColumnNameInCourses < ActiveRecord::Migration[7.0]
```

Comment:
Let's merge this and previous migration into the 1 migration, cos they're both are not in staging yet (Only locally)